attributes/default.rb

```diff
@@ -17,4 +17,9 @@
 # limitations under the License.
 #
 
-default['cloudmonkey']['version'] = nil
+default['sbp_cloudmonkey']['source'] = nil
+default['sbp_cloudmonkey']['version'] = '6.1.0'
+
+default['sbp_cloudmonkey']['binary'] = '/usr/local/bin/cmk'
+default['sbp_cloudmonkey']['create_cloudmonkey_link'] = true
```

Comment:
```suggestion
default['sbp_cloudmonkey']['create_symlink'] = true
```

I would also consider just removing this.